pkg/ui/react-app/src/themes/_bootstrap_dark.scss

```
@@ -37,6 +37,10 @@ body.bootstrap-dark {
   background-color: $body-bg; // 2
 }
 
+body.bootstrap-dark button{
+  color: white;
+}
```

```suggestion
body.bootstrap-dark button {
  color: $body-color;
}
```
We can use a variable for this instead of hardcoding white. I am not sure if the variable I suggested is the right one though.

---

Another option that we can consider (if it works)
```suggestion
body.bootstrap-dark button{
  color: inherit;
}
```

I am just spitting this out of my head so we need to test first if this works

---

Got it !!
I will check this now.

---

I tested it out 
Your first suggestion works Nice , I did thought about that earlier that we shouldn't hard code but i didn't knew how to implement.
Thanks!!
Now what should i do @onprem ?? Create a new PR or edit this here